tests/tutor/test_integration.py

```
@@ -0,0 +1,223 @@
+        )
+
+        assert package.completion_percentage == pytest.approx(50.0)
+        assert package.average_score == pytest.approx(0.7)
+        assert not package.is_complete()
+        assert package.get_next_topic() == "advanced"
+
+
+class TestBranding:
+    """Tests for branding/UI utilities."""
+
+    def test_tutor_print_success(self, capsys):
```

Ruff (0.14.13)

Unused method argument: `capsys`

(ARG002)